backend/tests/controllers/artefacts/test_artefacts.py

```diff
@@ -17,13 +17,53 @@
 # Written by:
 #        Omar Selo <omar.selo@canonical.com>
 #        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
+from datetime import timedelta
+
 from fastapi.testclient import TestClient
 from sqlalchemy.orm import Session
-from test_observer.data_access.models import ArtefactBuild, Environment, TestExecution
 
+from test_observer.data_access.models import ArtefactBuild, Environment, TestExecution
 from tests.helpers import create_artefact
 
 
+def test_get_latest_artefacts_by_family(db_session: Session, test_client: TestClient):
+    """Should only get latest artefacts and only ones that belong to given family"""
+    relevant_artefact = create_artefact(db_session, "edge", version="2")
+
+    old_timestamp = relevant_artefact.created_at - timedelta(days=1)
+    create_artefact(db_session, "edge", created_at=old_timestamp, version="1")
+    create_artefact(db_session, "proposed")
+
+    response = test_client.get("/v1/artefacts?family=snap")
```

Comment:
Is there a difference between `/v1/artefacts?family=snap` and `/v1/artefacts/?family=snap`?

Reply:
As far as I know they're both used interchangeably

Reply:
Shouldn't we pick one option and use it consistently?

Reply:
But I noticed that I can pass query parameters as an object. So I changed it to that